src/main/java/eu/europa/ted/efx/sdk2/EfxTemplateTranslatorV2.java

```diff
@@ -196,36 +202,45 @@ public EfxTemplateTranslatorV2(final MarkupGenerator markupGenerator,
    * Opens the indicated EFX file and translates the EFX template it contains.
    */
   @Override
-  public String renderTemplate(final Path pathname) throws IOException {
-
-    return renderTemplate(CharStreams.fromPath(pathname));
+  public String renderTemplate(final Path pathname, TranslatorOptions options) throws IOException {
+    return renderTemplate(CharStreams.fromPath(pathname), options);
   }
 
   /**
    * Translates the template contained in the string passed as a parameter.
    */
   @Override
-  public String renderTemplate(final String template) {
-    return renderTemplate(CharStreams.fromString(template));
+  public String renderTemplate(final String template, TranslatorOptions options) {
+    return renderTemplate(CharStreams.fromString(template), options);
   }
 
   @Override
-  public String renderTemplate(final InputStream stream) throws IOException {
-    return renderTemplate(CharStreams.fromStream(stream));
+  public String renderTemplate(final InputStream stream, TranslatorOptions options) throws IOException {
+    return renderTemplate(CharStreams.fromStream(stream), options);
   }
 
-  private String renderTemplate(final CharStream charStream) {
+  private String renderTemplate(final CharStream charStream, TranslatorOptions options) {
     logger.debug("Rendering template");
+    final long startTime = System.currentTimeMillis();
```

Comment:
It might be better to use the [StopWatch class from commons-lang3](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/time/StopWatch.html) (which is already a dependency).
Having one StopWatch instance for each "thing" measured would be clearer and help avoid mixing up the various start/end times.

Reply:
For the next iteration :)